CODEOWNERS

```diff
@@ -18,14 +18,14 @@
 /internal/server/openapi.go @ssoroka
 /internal/server/routes.go @ssoroka
 /internal/server/validator.go @ssoroka
+/internal/server/testdata/openapi3.json @ssoroka @hoyyeva @dnephin @jmorganca @pdevine
```

Comment:
I kept this because I think the file still serves the same purpose. To notify anyone interested in API changes.